uni/lib/view/locations/locations.dart

```diff
@@ -60,11 +62,6 @@ class LocationsPageViewState extends State<LocationsPageView> {
         Row(
           mainAxisAlignment: MainAxisAlignment.spaceBetween,
           children: [
```

Comment:
The search box is now a bit weird, placed at the left. Consider centering it and increasing the width.